search_service/proxy/elasticsearch.py

```diff
@@ -412,6 +413,8 @@ def parse_filters(filter_list: Dict) -> str:
             mapped_category = TABLE_MAPPING.get(category)
             if mapped_category is None:
                 LOGGING.warn(f'Unsupported filter category: {category} passed in list of filters')
+            elif item_list is '' or item_list == ['']:
```

Comment:
will it send empty from fe to search?

Reply:
I was testing it out through postman, and when I was manually sending empty values it was returning 500s.

Reply:
Got it, given the cors is already in https://github.com/lyft/amundsensearchlibrary/pull/89, could you remove the cors change in this pr?

Reply:
@feng-tao `item_list` currently does not risk being an empty string or a  list with an empty string when the request comes from frontend. However it could technically happen if someone introduces a bug that causes that.

Reply:
@feng-tao Sure!